### ISSUE TITLE
why i get a negative loss value?

### ISSUE THREAD

**Issue by FZH1802** [Maghoumi/pytorch-softdtw-cuda#32]
![微信截图_20231227192517](https://github.com/Maghoumi/pytorch-softdtw-cuda/assets/55188022/c9e0517c-a7ed-4f3c-a5a6-015a91a8245f)

**Comment by weirgmat** [comment]
Why? Me Too!

**Comment by ChunmingLi-SJTU** [comment]
Me too!! Asking  @Maghoumi  for help!

**Comment by Wave2689** [comment]
I get the same results when I train the data, so what happened? And what should I do?

**Comment by Maghoumi** [comment]
Please see #9, #10 and #27 

I think I should add a note about negative loss values in the readme file to avoid confusion in the future.

**Comment by mw66** [comment]
Also want to add: self-distance is also negative (not 0), and non-deterministic, try the example from the README, but between `x` and `x`:

```
loss = sdtw(x, x)

print(loss)
```

will get something like this:

```
tensor([-0.0106, -0.0804, -0.0233, -0.0286, -0.0194, -0.0570, -0.0206, -0.0297],
       device='cuda:0', grad_fn=<_SoftDTWCUDABackward>)
```
